src/v/pandaproxy/schema_registry/sharded_store.cc

```diff
@@ -678,6 +678,10 @@ ss::future<bool> sharded_store::is_compatible(
     co_return is_compat;
 }
 
+void sharded_store::check_mutable(force f) const {
+    _store.local().check_mutable(f).value();
+}
+
```

Comment:
> So this will throw a std exception of some sort with the error_info if no value, right? Could we put a note somewhere about what the shape of the exception will be?

Yes. It throws a specific exception the mechanics were intoduced here: https://github.com/redpanda-data/redpanda/commit/56f7e96cd1672bb24cd397c98737d03601d25b04